tests/L0/run_fused_layer_norm/test_fused_layer_norm.py

```diff
@@ -123,87 +129,87 @@ def _test_fused_rms_norm(
+        "batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient",
+        list(product((16, 65536), (True, False), (True,), (True,), (torch.float,), (True, False)))
     )
-    def test_rms_norm_mixed(self, batch_size, contiguous, elementwise_affine, mixed_fused, dtype):
-        self._test_fused_rms_norm(batch_size, contiguous, elementwise_affine, mixed_fused, dtype,
+    def test_rms_norm_mixed(self, batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient):
+        self._test_fused_rms_norm(batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient,
                                 bwd_thresholds=dict(rtol=2e-3, atol=2e-4))
     
     @common_utils.parametrize(
-        "batch_size, contiguous, elementwise_affine, mixed_fused, dtype",
-        list(product((16,), (True, False), (True,), (False,), (torch.half,)))
+        "batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient",
+        list(product((16,), (True, False), (True,), (False,), (torch.half,), (True, False)))
     )
-    def test_rms_norm_half(self, batch_size, contiguous, elementwise_affine, mixed_fused, dtype):
-        self._test_fused_rms_norm(batch_size, contiguous, elementwise_affine, mixed_fused, dtype,
+    def test_rms_norm_half(self, batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient):
+        self._test_fused_rms_norm(batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient,
                                 bwd_thresholds = dict(rtol=1.6e-2, atol=3e-3))
     
     @common_utils.parametrize(
-        "batch_size, contiguous, elementwise_affine, mixed_fused, dtype",
-        list(product((16,), (True, False), (True,), (False,), (torch.bfloat16,)))
+        "batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient",
+        list(product((16,), (True, False), (True,), (False,), (torch.bfloat16,), (True, False)))
     )
-    def test_rms_norm_bfloat16(self, batch_size, contiguous, elementwise_affine, mixed_fused, dtype):
-        self._test_fused_rms_norm(batch_size, contiguous, elementwise_affine, mixed_fused, dtype, 
+    def test_rms_norm_bfloat16(self, batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient):
+        self._test_fused_rms_norm(batch_size, contiguous, elementwise_affine, mixed_fused, dtype, memory_efficient, 
                                 fwd_thresholds=dict(rtol=1.6e-2, atol=3e-4), bwd_thresholds=dict(rtol=1.6e-2, atol=3e-2))
 
     @common_utils.parametrize(
-        "dtype, elementwise_affine",
-        list(product(autocast_dtypes, (True, False)))
+        "dtype, elementwise_affine, memory_efficient",
+        list(product(autocast_dtypes, (True, False), (True, False)))
     )
-    def test_autocast_fused_layer_norm(self, dtype, elementwise_affine):
+    def test_autocast_fused_layer_norm(self, dtype, elementwise_affine, memory_efficient):
```

Comment:
on V100 I saw the following failures

```
FAILED test_fused_layer_norm.py::TestFusedLayerNormCUDA::test_autocast_fused_layer_norm_float16_elementwise_affine_False_memory_efficient_True_cuda_float16 - AssertionError: Tensor-likes are not close!
FAILED test_fused_layer_norm.py::TestFusedLayerNormCUDA::test_autocast_fused_layer_norm_float16_elementwise_affine_True_memory_efficient_True_cuda_float16 - AssertionError: Tensor-likes are not close!
```

Reply:
Hi,

Are those the two tests referenced at the initial post? https://github.com/NVIDIA/apex/pull/1715#issue-1864532937

I have not modify those two tests yet.